src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -700,7 +700,26 @@ public class StripePaymentService : IPaymentService
                 DaysUntilDue = 1,
                 CollectionMethod = "send_invoice",
                 ProrationDate = prorationDate,
-            });
+            };
+
+            var customer = await new CustomerService().GetAsync(sub.CustomerId);
+            var taxRates = await _taxRateRepository.GetByLocationAsync(
+                new Bit.Core.Models.Table.TaxRate()
+                {
+                    Country = customer.Address.Country,
+                    PostalCode = customer.Address.PostalCode
+                }
+            );
+            var taxRate = taxRates.FirstOrDefault();
+            if (taxRate != null && !sub.DefaultTaxRates.Any(x => x.Equals(taxRate.Id)))
+            {
+                subUpdateOptions.DefaultTaxRates = new List<string>() 
```

Comment:
remove empty parens `()` at end of object constructor when using simplified object initialization (`{ ... }`). You can optionally pass in an explicit number `1` to the constructor args since it will only ever have a count of 1, therefore the list internals will not eat up more space in memory allocation/buffer size.